src/servers/ZoneServer2016/zonepackethandlers.ts

```diff
@@ -2256,6 +2256,10 @@ export class ZonePacketHandlers {
           "Parts may be required. Open vehicle loadout."
         );
         break;
+      case ItemUseOptions.UNPACK:
+        server.useAmmoBox(client, item);
```

Comment:
If there were one in the original game then yes

Reply:
There was